cmd/webpkgserver/README.md

```diff
@@ -52,6 +52,7 @@ in the repository root.
 [golang]: https://golang.org/doc/install
 [CanSignHttpExchanges]: https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#cross-origin-cert-req
 [DigiCert]: https://www.digicert.com/account/ietf/http-signed-exchange.php
+[Google]: https://cloud.google.com/blog/products/identity-security/automate-public-certificate-lifecycle-management-via--acme-client-api
```

Comment:
I recall there being a slightly different process for SXG certs, whereas it seems that blog post only documents TLS certs.

I think it's a different ACME discovery URL? Whatever it is, can you document it in this README?

Reply:
Added.